universe/vikram.hello/index.js

```diff
@@ -0,0 +1,194 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of vikram.hello
+//
+// Copyright 2021 undefined Vikram Nagapudi
+//                          vikramnagapudi2004@gmail.com
+// See LICENSE for details
+"use strict";
+const Tp = require('thingpedia');
+
+
+const tmdbAccess = "https://api.themoviedb.org/3/";
+const movieSearch = "search/movie?api_key=";
+const finalSearch = "&language=en-US&page=1";
+const topRated = "movie/top_rated?api_key=";
+const nowPlaying = "movie/now_playing?api_key=";
+
+
+module.exports = class MovieClass extends Tp.BaseDevice {
+    //constructor taken from Yelp index.js
+    constructor(engine, state){
+        super(engine, state);
+        this.uniqueId = "vikram.hello";
+    }
+
+    async get_movie (params, hints, env) {
+        // const queryURL = tmdbAccess + multiSearch + this.constructor.metadata.auth.api_key + finalSearch + realquery + "&page=1&include_adult=false";
+        let sortURL = '';
+        if (hints && hints.sort) {
+            if (hints.sort[0] === 'release_date' && hints.sort[1] === 'desc')
+                sortURL = tmdbAccess + nowPlaying + this.constructor.metadata.auth.api_key + finalSearch;
+            else if (hints.sort[0] === 'rating_score' && hints.sort[1] === 'desc')
+                sortURL = tmdbAccess + topRated + this.constructor.metadata.auth.api_key + finalSearch;
+        }
+        if (sortURL) {
+            const response1 = await Tp.Helpers.Http.get(sortURL);
+                let parsedResponse = JSON.parse(response1);
+                return Promise.all(parsedResponse.results.map(async (result) => {
+                    const castQuery = "https://api.themoviedb.org/3/movie/" + String(result.id) + "/credits?api_key=654083fa9e049ea234e4ae94d3e65774&language=en-US";
+                    let id = new Tp.Value.Entity(String(result.id), result.title);
+                    let oneDate = new Date(Date.now());
+                    if (String(result.release_date) != ''){
```

Comment:
This should be `!==`. Generally speaking, you should run `make lint` to catch all these errors right away. I won't be merging the skill if there are lint issues.